project/StdBits.scala

```diff
@@ -113,4 +115,31 @@ object StdBits {
       name.stripSuffix(".jar")
     } else file.getName
   }
+
+  def buildStdLibPackage(name: String, root: File, cacheFactory: sbt.util.CacheStoreFactory, log: sbt.Logger, defaultDevEnsoVersion: String) = Def.task {
+    log.info(s"Building standard library package for '$name'")
+    val prefix = "Standard"
+    val targetPkgRoot = root / "lib" / prefix / name / defaultDevEnsoVersion
+    val sourceDir = file(s"distribution/lib/$prefix/$name/$defaultDevEnsoVersion")
+    if (!sourceDir.exists) {
+      throw new RuntimeException("Invalid standard library package " + name)
+    }
+    LibraryManifestGenerator.generateManifests(
```

Comment:
Note that this and `fixLibraryManifest` are essentially copy pasted from `buildEngineDistribution` task because I assumed this was needed.